src/char.cr

```diff
@@ -69,6 +69,38 @@ struct Char
     ord - other.ord
   end
 
+  # Makes a new string by adding *str* to itself *times* times.
+  #
+  # ```
+  # '-' * 10 # => "----------"
+  # ```
+  def *(times : Int)
+    raise ArgumentError.new "negative argument" if times < 0
+
+    if times == 0 || bytesize == 0
```

Comment:
@RX14 I was still thinking of `String`, my bad - thanks for pointing that out! _just fix_-ed _it_ ™